Dockerfile-prebuilt

```diff
@@ -0,0 +1,14 @@
+ARG VERSION
+
+FROM iodide/pyodide-env:${VERSION}
+
+USER root
+
+COPY . pyodide
+
+RUN cd pyodide && make
+
+# this deletes the build results such that the resulting image can still be used for building pyodide
+# from source (including partial and customized builds). Due to the previous run of make, builds
+# executed with this image will be much faster than the ones executed with pyodide-env
+RUN rm -r build
```

Comment:
Are you sure this is needed? We could just remove the `build/package.json` and it should work I think. That way for packages depending on an existing dependency (say numpy) it wouldn't have to be rebuilt.

Reply:
I think it depends on how the image is supposed to be used. For example, I used it to build my package and copy everything from the build dir to the dir with frontend code. Generally, by performing a partial build after setting the PYODIDE_PACKAGES env var, I wouldn't expect any other packages in the build directory. Even if the unnecessary packages are filtered out in a subsequent build step (e.g. with npm build), it would be natural to have the contents of the build dir as artifacts (say in a ci job) and they should not be larger than required. I think partial builds will be the most common practice (at least for the purpose of frontend applications), since the entire pyodide stack is huge in size.

Also, rebuilding with this image is pretty fast (even numpy). But maybe I'm missing something here, I am not very familiar with your build process.

Reply:
Yes, you are right we can remove it. Rebuilding is fast as all the package build folder (`packages/*/build`) are still there).